docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,342 @@
+
+Second, select **Event Subscriptions** from the side and update the **Request URL**:
+
+![Event Subscriptions page](../assets/event-subscriptions-page.png "Event Subscriptions page")
+
+**3. Test your Slack app**
+
+Now we can test your Slack app by opening a Slack channel that your app has joined and saying “hello” (lower-case). Just like in the [Getting Started guide][getting-started-guide], your app should respond back:
+
+> 👩‍💻 hello<br/>
+> 🤖 Hey there @Jane!
+
+If you don’t receive a response, check your **Request URL** and try again.
+
+> 💡 **How does this work?**
+> The ngrok and Serverless commands are configured on the same port (default: 3000). When a Slack event is sent to your **Request URL**, it's received on your local machine by ngrok. The request is then forwarded to Serverless Offline, which emulates an AWS Lambda function event and triggers your Bolt app's receiver. 🛫🛬 Phew, what a trip!
+
+---
+
+### Deploy the app
+
+We'll use the Serverless Framework tools to provision, package, and deploy your app onto AWS Lambda. After your app is deployed, we'll update your Slack app's settings, and say "hello" to your app. ✨
+
+**1. Deploy the app to AWS Lambda**
+
+You can now deploy your app to AWS Lambda with the command:
+
+```shell
+npx serverless deploy
+# Serverless: Packaging service...
+# ...
+# endpoints:
+#   POST - https://atuzelnkvd.execute-api.us-east-1.amazonaws.com/dev/slack/events
+# ...
+```
+
+After your app is deployed, you'll be given an **endpoint**. Go ahead and copy this **endpoint** to use in the next section.
+
+> 💡 The **endpoint** should end in `/slack/events`.
+
+**2. Update your Slack app's settings**
+
+Now we need to use your AWS Lambda **endpoint** as your **Request URL**, which is where Slack will send events and actions.
+Head over to your [Slack app's settings](https://api.slack.com/apps) and select your app name. We'll update your **Request URL** in two locations.
+
+First, select **Interativity & Shortcuts** from the side and update the **Request URL**:
+
+![Interactivity & Shortcuts page](../assets/interactivity-and-shortcuts-page.png "Interactivity & Shortcuts page")
+
+Second, select **Event Subscriptions** from the side and update the **Request URL**:
+
+![Event Subscriptions page](../assets/event-subscriptions-page.png "Event Subscriptions page")
+
+**3. Test your Slack app**
+
+Your app is now deployed and Slack is updated, so let's try it out!
+
+Open a Slack channel that your app has joined and say "hello" (lower-case). Just like when we [ran the app locally](#run-the-app-locally), your app should respond back:
+
+> 👩‍💻 hello<br/>
+> 🤖 Hey there @Jane!
+
+**4. Deploy an update**
+
+As you continue building your Slack app, you'll need to deploy updates. Let's get a feel for this by updating your app to respond to a "goodbye" message.
+
+Add the following code to `app.js` ([source code on GitHub][getting-started-with-serverless-framework-app/app.js]):
```

Comment:
This link will work after the PR is merged, until then a preview of the link is at:
https://github.com/mwbrooks/bolt-js/blob/220-faas-deployment-guides/examples/getting-started-with-serverless-framework/app.js